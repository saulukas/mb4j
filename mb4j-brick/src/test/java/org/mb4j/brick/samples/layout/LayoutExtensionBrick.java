package org.mb4j.brick.samples.layout;

public class LayoutExtensionBrick extends LayoutBrick {

    public LayoutExtensionBrick() {
        this.content = new ContentBrick();
    }
}
