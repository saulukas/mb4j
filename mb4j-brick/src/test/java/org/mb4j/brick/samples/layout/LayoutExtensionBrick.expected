
   Header: Enterprise System

   ....
   ....   Some content from extending Brick.
   ....
   ....   Kind of removes need for separate layout/tiles mechanism.
   ....

   Footer: Â© 2013
